packages/core/src/EntityManager.ts

```diff
@@ -475,8 +475,11 @@ export class EntityManager<D extends IDatabaseDriver = IDatabaseDriver> {
       return entity;
     }
 
+    const meta = this.metadata.get(entityName);
```

Comment:
it is ok to use `find()!` here (faster than `get()`), as the validation check is already done few lines above (on 471)

```suggestion
    const meta = this.metadata.find(entityName)!;
```